1-mandatory/3-reading-list.js

```diff
@@ -16,11 +16,47 @@ In this style of testing it is typical to write out as strings exactly what you
 without using any variables or any logic like loops, template strings or if statements.
 */
 
-const books = [];
+const books = [
+  {
+    title: "1984",
+    author: "George Orwell",
+    alreadyRead: true,
+  },
+  {
+    title: "Ince Mehmed",
+    author: "Yasar Kemal",
+    alreadyRead: true,
+  },
+  {
+    title: "Deniz Kurdu",
+    author: "Jack London",
+    alreadyRead: false,
+  },
+  {
+    title: "Simyaci",
+    author: "Paulo Coelho",
+    alreadyRead: true,
+  },
+  {
+    title: "War and Peace",
+    author: "Tolstoy",
+    alreadyRead: false,
+  },
+];
 
 // exercise 1
-function logBooks() {}
+//function logBooks() {}
+const logBooks = books.map(book => `${book.title} by ${book.author}`)
 
+//console.log(logBooks)
+
+const logBooks2 = books.map((book) => {
+  if (book.alreadyRead == true) {
+    return `You've already read ${book.title} by ${book.author}`;
+  } else {
+    return `You still need to read \"${book.title}\" by ${book.author}.`
+  }
```

Comment:
here, you don't need the else statement and can follow early return patterrn
```suggestion
  if (book.alreadyRead == true) {
    return `You've already read ${book.title} by ${book.author}`;
  } 
  return `You still need to read \"${book.title}\" by ${book.author}.`
  
```